zinc/lib/Zinc_types.ml

```diff
@@ -35,10 +35,10 @@ type 'a zinc_instruction =
   | Num of Z.t
   | Add
   (* ASTs *)
-  | MakeRecord of label list
+  | MakeRecord of label
```

Comment:
```suggestion
  | MakeRecord of int
```
I know I suggested this but it was actually wrong

Reply:
done